ocaml/xapi/rpm.mli

```diff
@@ -56,6 +56,8 @@ module Pkg : sig
   val lte : Epoch.t -> string -> string -> Epoch.t -> string -> string -> bool
 
   val gte : Epoch.t -> string -> string -> Epoch.t -> string -> string -> bool
+
```

Comment:
This interface lacks documentation. What are all the strings for the comparison functions? Using labels could help to document them in code; or some comment is needed. I would have expected that the comparison functions work on `t` values.

Reply:
I added a few comments as mitigation in this PR. The best may be to refine the typing to abstract a `RpmVersion.t` for  epoch:version-release. Could do it in another PR.